### PR TITLE
base16384 2.2.2 (new formula)

### PR DESCRIPTION
**Pull request by fumiama**
- [x] Have you followed the [guidelines for contributing](https://github.com/Homebrew/homebrew-core/blob/HEAD/CONTRIBUTING.md)?
- [x] Have you ensured that your commits follow the [commit style guide](https://docs.brew.sh/Formula-Cookbook#commit)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/Homebrew/homebrew-core/pulls) for the same formula update/change?
- [x] Have you built your formula locally with `brew install --build-from-source <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Is your test running fine `brew test <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Does your build pass `brew audit --strict <formula>` (after doing `brew install --build-from-source <formula>`)? If this is a new formula, does it pass `brew audit --new <formula>`?

-----

This software has also been contributed to Debian and OpenWrt.

### CODE REVIEW

File: Formula/base16384.rb [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+
+  def install
+    system "cmake", "-S", ".", "-B", "build", *std_cmake_args
+    system "cmake", "--build", "build"
+    system "cmake", "--install", "build"
+  end
+
+  test do
+    assert_match "1234567890abcdefg",
+      shell_output("echo 1234567890abcdefg | #{bin}/base16384 -e - - | #{bin}/base16384 -d - -")
```
Comment (SMillerDev) [inline Formula/base16384.rb:19]: I think less pipes will make it easier to read
```suggestion
    hash = pipe_output("#{bin}/base16384 -e - -", "1234567890abcdefg")
    assert_match "1234567890abcdefg", pipe_output("#{bin}/base16384 -d - -", hash)
```

Comment (fumiama) [reply Formula/base16384.rb:19]: Thanks, I do not know much about the grammar.

Comment (SMillerDev) [reply Formula/base16384.rb]: No problem, I'm here to help

**Review by dawidd6 — APPROVED** [review]

**Review by chenrui333 — APPROVED** [review]

**Comment by chenrui333** [comment]
@fumiama, thanks for your contribution to Homebrew! :tada: 🥇

Without awesome contributors like you, it would be impossible to maintain Homebrew to the high level of quality users have come to expect. Thank you!!!!

**Pull request closed without merging**